_posts/2014-05-29-touristic-autism_continuous-deployment.markdown

```diff
@@ -29,10 +29,10 @@ OpenShift is one of the cloud deployment services offering such facilities. It i
       end
 {% endhighlight %}
   </div>
-If you are not using RVM or RailsInstaller then follow [this guide](https://www.openshift.com/developers/rhc-client-tools-install) (you may need to do "sudo gem install rhc").
+Se você não está usando o RVM ou o RailsInstaller então siga [esse guia](https://www.openshift.com/developers/rhc-client-tools-install) (você talvez precise executar o comando "sudo gem install rhc").
```

Comment:
Aqui por algum motivo não pegou o markdown:
![image](https://user-images.githubusercontent.com/4131432/66111655-96a39180-e5c9-11e9-9b25-cad3e302a478.png)

Você poderia dar uma investigada no por que disso? Talvez seja por que tem dois parênteses seguidos, por mim, podemos remover o segundo, se for o caso.